app/src/main/java/com/ayush/weatherapp/home/HomePresenterImpl.java

```diff
@@ -60,6 +61,9 @@
   }
 
   private void fetchCurrentLocation() {
+
+    LocationUtils.checkLocationServices(view.getContext());
```

Comment:
this is a void method and name looks/needs a boolean. If location is not provided a dialog is shown but still location request object is created.

Reply:
:ok: